src/pages/events/convert-field-values.md

```diff
@@ -0,0 +1,142 @@
+
+```php
+interface FieldConverterInterface
+{
+    /**
+     * Converts a field value
+     *
+     * @param mixed $value
+     * @param Event $event
+     * @return mixed
+     */
+    public function convert(mixed $value, Event $event);
+}
+```
+
+As an example, the `observer.catalog_product_save_after` event contains a top-level `visibility` field, which must contain an integer value. You want to convert these values to strings that match values on the external system. The following table describes these values.
+
+Commerce value | Converted value | Description
+--- | --- | ---
+1 | NOT_VISIBLE_INDIVIDUALLY | This product should not be displayed if it is part of a configurable product.
+2 | CATALOG_ONLY | This product appears in catalog listings, but not in searches.
+3 | SEARCH_ONLY | This product appears in searches, but not catalog listings.
+4 | CATALOG_AND_SEARCH | This product appears in catalog listings and searches. For most products, this is the default.
+
+In the following example, the `TestConverterVisibility` converter class updates the value of the `visibility` field to a string.
+
+```php
+<?php
+/**
+ * Copyright &copy; Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\AdobeCommerceEventsClient\Event;
+
+use Magento\AdobeCommerceEventsClient\Event\Filter\FieldConverterInterface;
+
+class TestConverterVisibility implements FieldConverterInterface
+{
+    /**
+     * Method used to convert field value
+     *
+     * @param mixed $value
+     * @return mixed
+     */
+    public function convert(mixed $value, Event $event): mixed
+    {
+        return 'CATALOG_AND_SEARCH';
+    }
+}
+```
+
+The default payload for this event includes the following:
+
+```json
+{
+   "event":{
+      "data":{
+         "value":{
+            "visibility":"4",
+        }
+      }
+   }
+}
+```
+
+The converter changes the payload to:
+
+```json
+{
+   "event":{
+      "data":{
+         "value":{
+            "visibility":"CATALOG_AND_SEARCH",
+        }
+      }
+   }
+}
+```
+
+## Register the converter
+
+You must configure a module's `io_events.xml` or root `app/etc/io_events.xml` file to update the required fields. You can also declare them in the system `config.php` file or add them using the CLI while subscribing to an event.
+
+### Command line
+
+The [`bin/magento events:subscribe --fields` command](commands.md#subscribe-to-an-event) defines the fields and converters to include in the payload of a subscribed event. The example command adds the `visibility` field and provides the path to the converter class. You can specify multiple fields in the same request.
+
+```bash
+bin/magento events:subscribe observer.catalog_product_save_after --fields="store_id" --fields='{"name":"visibility", "converter": "Magento\AdobeCommerceEventsClient\Event\TestConverterVisibility"}'`
+```
+### `config.php` file
+
+```php
```

Comment:
Example of config.php file it the event is subscribed using the bash command on [line 106](https://github.com/AdobeDocs/commerce-extensibility/pull/131/files#diff-36693f389686ca3991236ee3437b72b755c9770cb875d50f78f211d1c76b4ec6R106)